pom.xml

```diff
@@ -1867,7 +1875,7 @@
         <jdk>[1.8,)</jdk>
       </activation>
       <properties>
-        <javadoc.opts>-Xdoclint:none</javadoc.opts>
+        <doclint>${javadoc.doclint}</doclint>
```

Comment:
Here javadoc validating doclint was totally disabled, but semantics changed and this was not working anymore (replaced by simple `all`, `none` etc.)